FE/home-keeping-website/src/routes.jsx

```
@@ -123,13 +126,12 @@ function AppRoutes() {
         <Route path="/family/messages" element={<FamilyMessagesPage />} />
         <Route path="/family/jobs" element={<JobListOfFamily />} />
         <Route path="/family/support-requests" element={<FamilySupportRequestPage />} />
-        <Route path="/dashboard/support" element={<StaffSupportRequestPage />} />
         <Route path="/family/withdraw-details" element={<WithdrawRequestPage />} />
         {/* Admin Pages */}
         <Route path="/admin/dashboard" element={<AdminDashboardPage />} />
         <Route path="/admin/users" element={<AdminUserListPage />} />
         <Route path="/admin/services" element={<AdminServiceListPage />} />  // ✅ NEW SERVICE PAGE
-
+        <Route path="/admin/platform-fees" element={<ManagePlatformFeesPage />} />
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)